frontend/src/components/home/measurements/PeriodPicker.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { MonthPicker } from "../../controls/form-controls/MonthPicker";
+import { Moment } from "moment";
+import { t } from "../../utils";
+
+interface Props {
+  from: Moment;
+  to: Moment;
+  fromChange: (value: any) => void;
+  toChange: (value: any) => void;
+}
+
+const PeriodPicker = ({from, to, fromChange, toChange}: Props) => {
```

Comment:
Fyi

Hier was er mogelijkheid om een abstractie te introduceren: `periodChange` die de from-to samen neemt in 1 callback
from/to zou ook een `Range` kunnen zijn

Je moet daar natuurlijk altijd denken: maakt die abstractie me het makkelijker of net niet...

In dit geval:
- `Range` : it depends, zou moeten bekeken worden... omdat er nergens een Range bestaat heeft het waarschijnlijk geen zin
- `PeriodChange` : het is logischer om hier met 1 callback te werken: het bestaat uit twee controls maar is 1 component en een component heeft typisch 1 `onChange` callback.

Waarschijnlijk heb je waar je dit component gebruikt 2 callbacks die hetzelfde doen....